### PR TITLE
Initial code.

### PR DESCRIPTION
**Pull request by vitormcpedro**

### CODE REVIEW

File: yk_utils/files/configuration_files.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+"""Handling of configuration files"""
+import json
+
+
+def load_json_config(filename='./client_secrets.json'):
```
Comment (mlourencoeb) [inline yk_utils/files/configuration_files.py:5]: Can we remove this default and throw ValueError is filename is empty?

File: yk_utils/apis/face_authentication.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        """
+        self.api_url = api_url
+        self.api_key = api_key
+
+        self.status = None
+        self.message_class = None
+        self.message = None
+
+    @staticmethod
+    def allowed_base64_image(image_str):
```
Comment (mlourencoeb) [inline yk_utils/apis/face_authentication.py:23]: I would change image_str to image:str

File: yk_utils/apis/requests.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
+"""HTTP requests module.
+"""
+import requests
+from yk_utils.apis import BaseUrl, Key, YoonikApiException
+
+
+def request(method, url, data=None, json=None, headers=None, params=None):
```
Comment (mlourencoeb) [inline yk_utils/apis/requests.py:7]: typing missing in this method.

File: yk_utils/apis/yoonik_api_exception.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+"""YoonikApiException module
+"""
+
+
+class YoonikApiException(Exception):
+    """Custom Exception for the YooniK APIs."""
+    def __init__(self, status_code, message):
```
Comment (mlourencoeb) [inline yk_utils/apis/yoonik_api_exception.py:7]: typing missing in this method.

File: yk_utils/files/configuration_files.py (hunk 1 above)
Comment (mlourencoeb) [reply yk_utils/files/configuration_files.py:5]: typing missing in this method.

File: yk_utils/image/image_parser.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
+"""Image parser module.
+"""
+import os
+import base64
+
+
+def parse_image(image) -> str:
```
Comment (mlourencoeb) [inline yk_utils/image/image_parser.py:7]: Missing typing.

**Review by mlourencoeb — APPROVED** [review]

File: yk_utils/files/configuration_files.py (hunk 1 above)
Comment (vitormcpedro) [reply yk_utils/files/configuration_files.py:5]: Sure, will fix this :)

File: yk_utils/image/image_parser.py (hunk 5 above)
Comment (vitormcpedro) [reply yk_utils/image/image_parser.py:7]: Here image can have multiple types (an actual image as base64 string, a file path to an image or a file-like object)

**Pull request merged**